features/admin.alternative-login-identifier.v1/pages/alternative-login-identifier-page.tsx

```diff
@@ -407,10 +428,60 @@ const AlternativeLoginIdentifierInterface: FunctionComponent<AlternativeLoginIde
         if (!isAlphanumericUsername) {
             checkedClaims = checkedClaims.filter((item: string) => item !== ClaimManagementConstants.EMAIL_CLAIM_URI);
         }
+
+        return checkedClaims;
+    };
+
+    /**
+     * Checks if any claims need uniqueness scope update
+     */
+    const needsUniquenessScopeUpdate = (checkedClaims: string[]): boolean => {
+        return checkedClaims.some((claimURI: string) => {
+            const claim: Claim = availableClaims.find((c: Claim) => c.claimURI === claimURI);
+
+            return !claim.uniquenessScope || claim.uniquenessScope !== UniquenessScope.ACROSS_USERSTORES;
```

Comment:
Checking claim.uniquenessScope !== UniquenessScope.ACROSS_USERSTORES is sufficient since undefined or null will already fail this comparison.
Check other places as well.

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7106/commits/3088d327c477257341387a57516db4d9929fbfaa